packages/shared/src/components/onboarding/OnboardingHeader.tsx

```diff
@@ -59,13 +63,18 @@ export const OnboardingHeader = ({
             position={LogoPosition.Relative}
             linkDisabled
           />
-          {showButtonOnScreens.includes(activeScreen) && (
+          {showCreateFeedButton.includes(activeScreen) && (
             <CreateFeedButton
-              onClick={onClickCreateFeed}
+              onClick={onClick}
               customActionName={customActionName}
               activeScreen={activeScreen}
             />
           )}
+          {showNotNowButton.includes(activeScreen) && (
+            <Button onClick={onClick} variant={ButtonVariant.Tertiary}>
+              {customActionName}
+            </Button>
+          )}
```

Comment:
why do we need this, the above already supports customActionName?
What's different?

If it's just variant, we should make that dynamic in the above one I feel like.